cockatrice/src/userlist.cpp

```diff
@@ -110,8 +115,8 @@ BanDialog::BanDialog(const ServerInfo_User &info, QWidget *parent)
 
 void BanDialog::okClicked()
 {
-    if (!nameBanCheckBox->isChecked() && !ipBanCheckBox->isChecked()) {
-        QMessageBox::critical(this, tr("Error"), tr("You have to select a name-based or IP-based ban, or both."));
+    if (!nameBanCheckBox->isChecked() && !ipBanCheckBox->isChecked() && !idBanCheckBox->isChecked()) {
```

Comment:
I think he means that it can be a name ban, ip ban, id ban, name+ip, name+id, etc etc. rather than only having one or all of them as options.

That is, it can be any combination such that `a || b || c === true`, not a combination such that only `a`, `b`, or `c` is true or `a && b && c` is true, like the text description seems to imply.